values/gitea/gitea.gotmpl

```diff
@@ -110,18 +112,31 @@ gitea:
       DOMAIN: {{ $giteaDomain }}
       ROOT_URL: "https://{{ $giteaDomain }}/"
       DISABLE_SSH: true
-
+      HTTP_PORT: 3000
   metrics:
     enabled: true
     serviceMonitor:
       enabled: true
-      additionalLabels:
-        prometheus: system
+      relabelings:
```

Comment:
I am curious how have you discovered that metrics need to be relabeled. There is interesting lesson to be learned :)

Reply:
It is essentially a replacement for `additionalLabels` which is not supported anymore. The `sourceLabels` setting is just a dummy value in this case, but generally `relabelings` are a more generic method that also allow for updating existing labels using workload variables.